homeassistant/components/homekit/type_locks.py

```diff
@@ -54,7 +55,8 @@ def set_state(self, value):
         params = {ATTR_ENTITY_ID: self.entity_id}
         if self._code:
             params[ATTR_CODE] = self._code
-        self.hass.services.call(DOMAIN, service, params)
+        super().set_state(DOMAIN, service, params, service)
```

Comment:
In this case it is "service" twice as for "locks" what is passed as the message is the same as what is in service. But for other accessories it would be different.
What I can do is make it optional and if not provided then use service instead.

Reply:
Optional is a good start. It depends how we will structure the `message` then, with which if clauses ;)

Reply:
Changed it to optional and if not provided then I will take what was provided from service instead.